templates/usr/local/sbin/secure_owncloud.sh.j2

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# This script sets the recommended modes and ownership for your ownCloud 
+# directories and files
+# https://doc.owncloud.org/server/9.0/admin_manual/installation/installation_wizard.html#strong-perms-label
+
+# {{ ansible_managed }}
+
+[[ "$UID" != 0 ]] && { echo "You must be root"; exit 1; }
+
+set -eu -o pipefail
+
+ocpath='{{ owncloud_home }}'
```

Comment:
Looking at this script, in the back of my mind there's a thought - what happens if Ansible templates this variable with an empty string? Perhaps this could be used to make it more reliable:
```
ocpath='{{ owncloud_home }}:-/var/www/owncloud'
```
IIRC, `/var/www/owncloud` is the ownCloud home path in the .`deb` package deployment.